src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1900,7 +1903,9 @@ def aks_create(cmd, client, resource_group_name, name, ssh_key_value,  # pylint:
                yes=False):
     _validate_ssh_key(no_ssh_key, ssh_key_value)
     subscription_id = get_subscription_id(cmd.cli_ctx)
-    if not dns_name_prefix:
+    if dns_name_prefix and fqdn_subdomain:
+        raise CLIError('--dns-name-prefix and --fqdn-subdomain cannot be used at same time')
```

Comment:
In `azure-cli` repo, please use [new error types](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md#error-type) instead of `CLIError`. Here you can use `MutuallyExclusiveArgumentError`.

Reply:
thanks, fixed.